ui/app/src/services/publishing.tsx

```diff
@@ -0,0 +1,5 @@
+import { get } from "../utils/ajax";
+
+export function fetchPackages (siteId: string) {
+  return get(`/studio/api/2/publish/packages?siteId=${siteId}`, {'X-XSRF-TOKEN': '060f063c-7812-4426-abfa-a1169d1e300c'})
```

Comment:
Move the mockup XSRF token header to the App.tsx since that file is pretty much for that sort of stuff the way we use it.

Reply:
using setRequestForgeryToken();